server/utils/adjudicationFilterHelper.ts

```diff
@@ -19,14 +19,14 @@ const error: { [key in ErrorType]: FormError } = {
 export type UiFilter = {
   fromDate: string
   toDate: string
-  status: ReportedAdjudicationStatus
+  status: ReportedAdjudicationStatus[]
 }
 
 export const uiFilterFromRequest = (req: Request): UiFilter => {
   return {
     fromDate: req.query.fromDate as string,
     toDate: req.query.toDate as string,
-    status: req.query.status as ReportedAdjudicationStatus,
+    status: req.query.status as ReportedAdjudicationStatus[],
```

Comment:
This needs to be `status: req.query.status as ReportedAdjudicationStatus`
As I don't think the query value is ever an array?

Reply:
See previous comment